### PR TITLE
Remove unused import patterns, fix project for Django > 1.8

### PR DESCRIPTION
**Pull request by tonythomas01**
The import was showing up error on later Django versions (Now its 1.11+)

### CODE REVIEW

File: tox.ini [hunk 1]
```diff
@@ -1,10 +1,11 @@
...
-envlist = {py27,py34,py35}-django{18,19}
+envlist = {py27,py34,py35}-django{18,19,110}
 
 [testenv]
 deps =
     pytest
     pytest-django
     django18: Django==1.8.10
     django19: Django==1.9.3
+    django110: Django==1.10.4
```
Comment (jakeharding) [inline tox.ini]: Please add the latest Django version also: 1.11.1 I believe.

Comment (tonythomas01) [inline tox.ini]: I havent personally tried it with Django=1.11.1. Let me get travis run it on 1.10.4, and if that goes green - will add for sure.

**Comment by tonythomas01** [comment]
> Could edit this pull request so it will be merged into develop?
 
Ah. missed that :-(

File: tox.ini (hunk 1 above)
Comment (jakeharding) [inline tox.ini]: I have a branch that takes care of most it if you want to add 1.11.1 and change the base to develop we can merge it.

Comment (tonythomas01) [inline tox.ini]: I am trying to fix the issues with 1.10 now. Looks like some `reverse` imports from somewhere else post 1.10.

**Comment by tonythomas01** [comment]
Alright. Let me push this to the development branch.

**Comment by tonythomas01** [comment]
If the tests pass, I think we are done here.

**Comment by jakeharding** [comment]
Thanks!

**Pull request merged**